README.md

```diff
@@ -187,18 +182,77 @@ This will be resolved from the first of:
 
 <br/>
 
+
+## FAQ
+
+#### I've run `git machete discover`... but the branch layout I see in `.git/machete` doesn't exactly match what I expected. Am I doing something wrong?
+
+[//]: # (For how to find Medium header anchors, see https://www.freecodecamp.org/news/how-to-link-to-a-specific-paragraph-in-your-medium-article-2018-table-of-contents-method-e66595fea549/)
```

Comment:
Every time I do anything with Medium, I _discover_ (no pun intended) that it sucks even more than I thought last time.

Yes, Medium does NOT have regular `<a>` named anchors in headers:

![image](https://user-images.githubusercontent.com/3383210/180987013-85ffd035-0365-4d88-88cf-4927d1dbf06e.png)

At least there's some way around that... copy the id from `h1` 😅